recipes/snappy/all/conandata.yml

```diff
@@ -1,7 +1,14 @@
 sources:
+  "1.1.9":
+    url: "https://github.com/google/snappy/archive/1.1.9.tar.gz"
+    sha256: "75c1fbb3d618dd3a0483bff0e26d0a92b495bbe5059c8b4f1c962b478b6e06e7"
   "1.1.8":
-    url: https://github.com/google/snappy/archive/1.1.8.tar.gz
-    sha256: 16b677f07832a612b0836178db7f374e414f94657c138e6993cbfc5dcc58651f
+    url: "https://github.com/google/snappy/archive/1.1.8.tar.gz"
+    sha256: "16b677f07832a612b0836178db7f374e414f94657c138e6993cbfc5dcc58651f"
   "1.1.7":
-    url: https://github.com/google/snappy/archive/1.1.7.tar.gz
-    sha256: 3dfa02e873ff51a11ee02b9ca391807f0c8ea0529a4924afa645fbf97163f9d4
+    url: "https://github.com/google/snappy/archive/1.1.7.tar.gz"
+    sha256: "3dfa02e873ff51a11ee02b9ca391807f0c8ea0529a4924afa645fbf97163f9d4"
+patches:
+  "1.1.9":
+    - patch_file: "patches/1.1.9-fix-inlining-failure-functino-body-can-be-overwritten-at-link-time.patch"
```

Comment:
Wow, maybe reduce size of filename? 😄

Reply:
Hey, I don't work with DOS anymore :rofl:
Is `fxinline.dif` ok? :stuck_out_tongue: